### ISSUE TITLE
Windows only?

### ISSUE THREAD

**Issue by soundsleit** [SlicerIGT/SlicerSoundControl#8]
Hi - new to 3d slicer. I'm working with a team on a sonification project in conjunction with some MRI scans.  I've been fiddling around with SlicerSoundControl, being a long time Pd programmer.  However, I work on MacOS, and it looks to me like the plugin is geared for Windows only, yes?

**Comment by lassoan** [comment]
The module is pure Python, works on all platforms. You need to install [pure-data](http://puredata.info/) to generate the sound - it is available on Windows/Linux/Mac.

**Comment by soundsleit** [comment]
Yes - I’m very familiar with Pd.   But the python code seems to have PC-specific file paths that don’t work on MacOS.

**Comment by soundsleit** [comment]
PS:  this is the error we are getting when we try to run the plug-in:
  
    self.oscClient = OSC.OSCClient()
AttributeError: module 'OSC' has no attribute 'OSCClient'

We are using Pd 0.51-1 (vanilla - since extended is no longer maintained...)

**Comment by lassoan** [comment]
Probably this error is due to Python2/3 differences in relative imports. The module was developed when Slicer used Python2 but now it uses Python3. You can fix the issue by modifying OpenSoundControl\OSC\__init__.py (maybe changing its content to `from .OSC import *` is sufficient).

However, now Slicer can also install any Python package from PyPI, so if you have any trouble with making the bundled OpenSoundControl\OSC\OSC.py interface work then you can install [python-osc](https://pypi.org/project/python-osc/) or similar packages and use that (`pip_install('python-osc')`).
